src/main/java/org/kuali/kfs/module/ar/businessobject/LetterOfCreditBillingPeriod.java

```diff
@@ -56,5 +56,15 @@ protected Date determineStartDateByFrequency() {
         LOG.info("determineStartDateByFrequency: returning lastBilledDate =" + lastBilledDate);
         return lastBilledDate;
     }
+    
+    /*
+     * CU Customization
```

Comment:
since the other places in this PR references CU-MOD instead of CU Customization,  I think we may want to be consistent in these comments, to make it easier to find things